### PR TITLE
Gateway API blog post

### PR DESCRIPTION
**Pull request by frankbu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by howardjohn** [comment]
LGTM!

File: content/en/blog/2022/getting-started-gtwapi/index.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+configurations that for large applications should be managed by different teams and in different namespaces.
+Also, by trying to draw a common denominator across
+different HTTP proxies, Ingress is only able to support the most basic HTTP routing and ends up pushing
+every other feature of modern proxies into non-portable annotations.
+
+To overcome Ingress' shortcomings, Istio introduced its own configuration API for ingress traffic management.
+With Istio's API, the client-side representation is defined using an Istio Gateway resource, with L7 traffic
+moved to a VirtualService, not coincidentally the same configuration resource used for routing traffic between
+services inside the mesh. Although the Istio API provides a good solution for ingress traffic management
+for large-scale application, it is unfortunately an Istio-only API. If you are using a different service
```
Comment (ericvn) [inline content/en/blog/2022/getting-started-gtwapi/index.md:31]:
```suggestion
for large-scale applications, it is unfortunately an Istio-only API. If you are using a different service
```

File: content/en/blog/2022/getting-started-gtwapi/index.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+To use it, you'll need to used the
+[experimental version](https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard)
+of the Gateway API CRDs, instead of the standard Beta version we installed above, but otherwise, you're ready to go.
+Check out the Istio [request routing task](/docs/tasks/traffic-management/request-routing/)
+to get started.
+
+## Summary
+
+In this article, we've seen how a light-weight minimal install of Istio can be used to provide a Beta-quality implementation
+of the new Kubernetes API for cluster ingress traffic control. For Istio users, the Istio implementation also lets
```
Comment (ericvn) [inline content/en/blog/2022/getting-started-gtwapi/index.md:285]:
```suggestion
of the new Kubernetes Gateway API for cluster ingress traffic control. For Istio users, the Istio implementation also lets
```

**Review by ericvn — APPROVED** [review]
Thanks Frank.

**Review by kfaseela — APPROVED** [review]

**Pull request merged**